pages/Documents/MessagingChannels/ProactiveMessaging/api.md

```diff
@@ -73,15 +73,34 @@ Note: Proactive Messaging can be leveraged using Proactive 2.0 API or [Web Tool]
 {
     "campaignName": "TestProactiveAPI",
     "skill": "sales",
-    "templateId": "943679028015322",
```

Comment:
can we keep the examples separate for old version vs new version ?

Reply:
@mkumari-lp should we try this ?
--> new version examples
** Deprecated
--> old version examples